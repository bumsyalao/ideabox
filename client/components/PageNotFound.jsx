import React from 'react';

const PageNotFound = () => (
  <div className="bkg-image">
    <h4 className="white-text"> Error 404: Page Not Found </h4>
  </div>

);

export default PageNotFound;
