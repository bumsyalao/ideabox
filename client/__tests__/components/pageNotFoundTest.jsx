/* global expect test */
import React from 'react';
import { shallow, configure } from 'enzyme';
import toJson from 'enzyme-to-json';
import adapter from 'enzyme-adapter-react-16';
import PageNotFound from '../../components/PageNotFound';

configure({ adapter: new adapter() });

describe('Hompeage component', () => {
  const component = shallow(
    <PageNotFound />
  );
  it('should match the PageNotFound component snapshot', () => {
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
});
