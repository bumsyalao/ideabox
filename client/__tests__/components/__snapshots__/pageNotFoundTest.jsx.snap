// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hompeage component should match the PageNotFound component snapshot 1`] = `
<div
  className="bkg-image"
>
  <h4
    className="white-text"
  >
     Error 404: Page Not Found 
  </h4>
</div>
`;
